testsuite/MDAnalysisTests/transformations/test_translate.py

```diff
@@ -34,28 +34,116 @@
 def translate_universes():
     # create the Universe objects for the tests
     reference = make_Universe(trajectory=True)
-    transformed = make_Universe(trajectory=True)
-    transformed.trajectory.ts.dimensions = np.array([372.,373.,374.,90,90,90])
+    transformed = make_Universe(['masses'], trajectory=True)
```

Comment:
You may want to assign masses so that the center of mass and the center of geometry are different.

Reply:
using the `['masses'}` extra in `make_Universe` generates masses for the atoms. The centers are different for the transformed universe